src/menu/menu.c

```diff
@@ -67,6 +67,13 @@ static void menu_init (boot_params_t *boot_params) {
 
     sound_init_default();
 
+    // Ensure that any joypad rumble pak is in a good state when loading the menu.
+    // this can be especially important if a previous ROM has crashed or the console
+    // was reset and left it in an active state.
```

Comment:
is this comment really necessary? I'd say the function name `joypad_set_rumble_active` says exactly what it does
keep comments for really confusing stuff that's not clear at first

Reply:
Not really, but was helpful to me. I will remove it (since is saved in this PR if required).

Reply:
```suggestion

```